python/nix_csi/subprocessing.py

```
@@ -60,24 +68,38 @@ async def run_console(*args, log_level: int = logging.DEBUG):
-    await asyncio.gather(
-        stream_output(proc.stdout, stdout_data),
-        stream_output(proc.stderr, stderr_data),
-        proc.wait(),
-    )
+        try:
+            async for line in stream:
+                decoded = line.decode().strip()
+                buffer.append(decoded)
+                combined_data.append(decoded)
+                logger.log(log_level, decoded)
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)